python/tchannel/tornado/util.py

```diff
@@ -52,3 +53,21 @@ def print_arg(request, index):
                 print chunk
     except Exception:
         pass
+
```

Comment:
I'm pretty opposed to this whole file. The `print_arg` and `get_arg` methods seem unnecessary now that we have methods on the request, and these things could all be defined inline.